### PR TITLE
648 start to fill in services under constructor

### PR DESCRIPTION
**Pull request by DrewSadler01**

### CODE REVIEW

**Comment by Woodlnsoul** [comment]
This is probably not near ready, but has a framework down. I will add comments in the next commit, I just wanted to get something up to look at.

File: src/java/edu/slu/tpen/transfer/JsonLDExporter.java [hunk 1]
```diff
@@ -79,6 +79,26 @@ public JsonLDExporter(Project proj, User u, String profile) throws SQLException,
...
+               services.put("header", buildNoneLanguageMap("Login for image access"));
+               services.put("description", buildNoneLanguageMap("Agreement requires an open T-PEN session to view images"));
+               services.put("confirmLabel", buildNoneLanguageMap("Login"));
+               services.put("failureHeader", buildNoneLanguageMap("T-PEN Login Failed"));
+               services.put("failureDescription",buildNoneLanguageMap("<a href=\"http://t-pen.org/TPEN/about.jsp\">Read Agreement</a>"));
+
+                Map<String, Object> logout = new LinkedHashMap<>();
+                logout.put("@id", "http://t-pen.org/TPEN/login.jsp");
+                logout.put("profile", "http://iiif.io/api/auth/1/logout");
+                logout.put("label", "End T-PEN Session");
```
Comment (markskroba) [inline src/java/edu/slu/tpen/transfer/JsonLDExporter.java:97]: This should also use buildNoneLanguageMap()

File: src/java/edu/slu/tpen/transfer/JsonLDExporter.java [hunk 2]
```diff
@@ -79,6 +79,26 @@ public JsonLDExporter(Project proj, User u, String profile) throws SQLException,
                manifestData.put("label",buildNoneLanguageMap(proj.getProjectName()));
                manifestData.put("metadata", getMetadataAsJSON(projID, profile));
               
+               Map<String, Object> services;
+           services = new LinkedHashMap<>();
+               services.put("id","http://t-pen.org/TPEN/login.jsp");
```
Comment (markskroba) [inline src/java/edu/slu/tpen/transfer/JsonLDExporter.java:84]: From https://iiif.io/api/presentation/3.0/#services this should be `@id`

File: src/java/edu/slu/tpen/transfer/JsonLDExporter.java [hunk 3]
```diff
@@ -79,6 +79,26 @@ public JsonLDExporter(Project proj, User u, String profile) throws SQLException,
...
+               services.put("type",buildNoneLanguageMap("ImageService3"));
```
Comment (markskroba) [inline src/java/edu/slu/tpen/transfer/JsonLDExporter.java:85]: From https://iiif.io/api/presentation/3.0/#services this should be `type` and a value for it a string, not language map

**Review by markskroba — CHANGES_REQUESTED** [review]

**Review by thehabes — APPROVED** [review]
Looks good, confirm with Auth API documentation.

**Review by markskroba — APPROVED** [review]

**Pull request merged**